packaging/ios/build.sh

```diff
@@ -9,6 +9,7 @@ TOP=$(pwd)
 SUBSURFACE_SOURCE=${TOP}/../../../subsurface
 IOS_QT=${TOP}/Qt
 QT_VERSION=$(cd ${IOS_QT}; ls -d 5.*)
+QT_VERSION=5.10.0
```

Comment:
???

Reply:
Same as above. BTW I have to do this as your clever detection thing above does not work for me (Qt wants to install in a directory 5.10.0 for some reason).

Reply:
Can we fix the "clever detection" instead?